boba-chain-ops/README.md

```diff
@@ -0,0 +1,27 @@
+# boba-chain-ops
+
+This package performs state regenesis. It takes the following input:
+
+1. An `alloaction.json` file that contains a list of pre-allocated accounts.
+2. A `genesis.json` file that contains the genesis block configuration.
+3. A list of addresses that transacted on the network prior to this past regenesis.
+4. A list of addresses that performed approvals on prior versions of the OVM ETH contract.
+5. A list of msg information that stores the cross domain message from L2 to L1
+
+It creates an initialized Bedrock erigon database as output. It does this by performing the following steps:
+
+1. Create genesis (types.Genesis) from `genesis.json` and `allocation.json` in memory.
+2. Set up the proxy contract for the predeploy contracts and update the bytecode of the predeploy contracts.
+3. Migrate Boba legacy proxy implementation contract the new proxy contract and delete the slots
```

Comment:
when you mean migrate and delete slots, you mean on the new proxy contract right?

Reply:
The slots of our old implementation contracts are deleted. This means that the slots of `0x4200000000000000000000000000000000000021`, `0x4200000000000000000000000000000000000022` and `0x4200000000000000000000000000000000000025` are deleted. The bytecode is replaced by the proxy contract and the admin slot is added.